src/goSuggest.ts

```diff
@@ -346,6 +346,7 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 	private getMatchingPackages(word: string, suggestionSet: Set<string>): vscode.CompletionItem[] {
 		if (!word) return [];
 		let completionItems = [];
+		const rootPath = vscode.workspace.rootPath.slice(getCurrentGoPath().length + 5);
```

Comment:
This will fail in a few cases
- Multi-root mode. To fix this you can use `vscode.workspace.getWorkspaceFolder(fileUri)`
- The file being edited is not part of the folder opened in VS Code or the file alone has been opened in VS Code. No folders. In this case `vscode.workspace.getWorkspaceFolder(fileUri)` would not return a workspace, so you can fall back to the parent folder of the file.
- The file being edited is not part of the gopath
- The `getCurrentGopath` returns multiple gopaths like `projectA:projectB` or `projectA;projectB`. To solve this you can use `getCurrentGoWorkspaceFromGOPATH` method from `goPath.ts`